website/docs/docs/dbt-versions/core-upgrade/03-upgrading-to-v1.6.md

```diff
@@ -85,6 +85,20 @@ More consistency and flexibility around packages. Resources defined in a package
 [**Project dependencies**](/docs/collaborate/govern/project-dependencies): Introduces `dependencies.yml` and dependent `projects` as a feature of dbt Cloud Enterprise. Allows enforcing model access (public vs. protected/private) across project/package boundaries. Enables cross-project `ref` of public models, without requiring the installation of upstream source code.
 :::
 
+### Deprecated functionality
+
+The ability for installed packages to override built-in materializations without explicit opt-in from the user is being deprecated.
+
+- Overriding a built-in materialization from an installed package raises a deprecation warning.
+- Using a custom materialization from an installed package does not raise a deprecation warning.
+- Using a built-in materialization package override from the root project via a wrapping materialization is still supported. For example:
+
+  ```
+  {% materialization view, default %}
+  {{ return(my_cool_package.materialization_view_default()) }}
+  {% endmaterialization %}
+  ```
```

Comment:
wonder if this "deprecation notice" is worth capturing in a snippet/partial/reusable

Reply:
It's only going in the migration guides. When we actually deprecate it in 1.9 I'll see how extensive it is. Snippet might be a good call then.